mscore/playpanel.ui

```diff
@@ -479,24 +487,60 @@
           <height>64</height>
          </size>
         </property>
+        <property name="maximumSize">
+         <size>
+          <width>128</width>
+          <height>512</height>
+         </size>
+        </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="toolTip">
-         <string>Metronome volume</string>
+         <string>Relative tempo</string>
         </property>
         <property name="accessibleName">
-         <string>Metronome Volume</string>
+         <string>Relative Tempo to 120 beats per minute</string>
```

Comment:
That is more a toolTip than an accessibilty info, so better swap those 2

Reply:
Thanks. Not quite sure what was going on. And the relative to "120" wouldn't be true anyway given how the Play Panel currently works. Fixed.